README.md

```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+# Azure Container Compute Upstream Projects
+
+This list of projects is maintained by the Azure container compute upstream team (TODO: how to contact the team). This list is inteded to help you make informed decisions about what projects to use (or not use) in the context of your goals (e.g. proof of concept vs. production). To make this decision you need to consider your goals, your need for formal support, the project's [maturity](#Maturity), governance, version level, and your willingness to work in open source.
+
+## Support
+
+Projects listed on this page are open source that Microsoft maintain or contribute to. These projects are [**NOT** covered by the Microsoft Azure support policy](https://support.microsoft.com/en-us/help/2941892/support-for-linux-and-open-source-technology-in-azure). To get help please search open issues using the links. If your issue isn't already represented, please open a new one. Consider that if you consume one of these projects as a part of a Microsoft or Azure product or service, you may be eligible for [support under that product](https://support.microsoft.com/en-us/hub/4343728/support-for-business).
+
+## Project list
+
+| Project Area | Project & (artifacts) | Goal | Project State & <br> API Version | Communication | Use on Azure |
+|---|---|---|---|---|---|
+| Kubernetes Cluster Management |  |  |  |  |
+|  | [AKS Engine](https://github.com/Azure/aks-engine) <br> ([releases](https://github.com/Azure/aks-engine/releases)) | Self-managed clusters on Azure | Azure: incubating <br> API: N/A | [#aks-engine-users](https://kubernetes.slack.com/archives/CU3N85WJK) <br> [GitHub issues](https://github.com/Azure/aks-engine/issues) | [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/) <br> [Azure Stack Hub](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-kubernetes-aks-engine-overview) |
```

Comment:
Now we need to document the progression of each project through the graduation process

Reply:
For lists like AKS and Azure Stack Hub, maybe use a bullet list as the break is getting lost